src/config/axiosConfig.ts

```diff
@@ -7,7 +7,9 @@ export const client = Axios.create({
 client.interceptors.request.use(
   async (req) => {
     if (!req.headers?.Authorization) {
-      const token = (await AsyncStorage.getItem('token')) ?? '';
+      const token =
+        (await AsyncStorage.getItem('token')) ??
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjUxLCJyb2xlIjoiRWxkZXJseSIsImlhdCI6MTY0ODMxMTQxMywiZXhwIjoxNjQ4OTE2MjEzfQ.An-AYuAEVkgraP3ozrc2sYchpNpfhrOu101Tg32bVRs';
```

Comment:
Is this default token due to others' PR? If not, please remove it.

Reply:
I remember this being in PR #26 So I will just approve this and wait for fix in that PR

Reply:
Noted I will remove it. Kinda sleep last night, sorry lol.